easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_2016_composer_edition_for_cpp_2016.0.109.tgz']
```

Comment:
please avoid copy-pasting the version number in multiple places, use the `%(version)s` template instead (see also http://easybuild.readthedocs.org/en/latest/Writing_easyconfig_files.html#easyconfig-param-templates)

```python
sources = ['parallel_studio_xe_2016_composer_edition_for_cpp_%(version)s.tgz']
```